setup.py

```diff
@@ -41,9 +40,14 @@
     packages=find_packages("src"),
     package_dir={"": "src"},
     license="BSD",
-    long_description=open("README.md").read(),
+    long_description=open("README.md")
+    .read()
+    .replace(
```

Comment:
I thought this issue is only due to scale-factor. For other projects, badges seem to work ok when the scaling was removed and we don't have scaling here. When I locally test (after `python -m build`)
```
twine check dist/alchemlyb-2.3.1+5.g911ce53*
Checking dist/alchemlyb-2.3.1+5.g911ce53-py3-none-any.whl: PASSED
Checking dist/alchemlyb-2.3.1+5.g911ce53.tar.gz: PASSED
```
it looks fine without removing the badges.

What was your specific reason for removing the badges?

Reply:
Did you get an error from pypi?

The other linked issues seem to say that twine check passes but then pypi rejects, although I don't quite understand why as they are saying that they don't allow inclusion of local files whereas our badges are all links.